flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/batch/sql/SingleRowJoinTest.scala

```diff
@@ -187,7 +187,7 @@ class SingleRowJoinTest extends TableTestBase {
           unaryNode(
             "DataSetCalc",
             batchTableNode(1),
-            term("select", "0 AS $f0")),
+            term("select", "")),
```

Comment:
This looks wrong. Do we support empty rows? Could you verify this with an IT case?

Reply:
Year, this is right. Besides, this would bring performance improvement, as select() is Row(0) while select(0 as xx) is Row(1). The data will be smaller.
I will add the IT case in the coming update.